packages/roosterjs-editor-plugins/lib/plugins/ImageEdit/editInfoUtils/getTargetSizeByPercentage.ts

```diff
@@ -0,0 +1,18 @@
+import ImageEditInfo from '../types/ImageEditInfo';
+
+/**
+ * @internal
+ * Get target size of an image with a percentage
+ * @param editInfo
+ * @param percentage
+ * @returns [width, height] array
+ */
+export default function getTargetSizeByPercentage(
+    editInfo: ImageEditInfo,
+    percentage: number
+): number[] {
```

Comment:
can we return an object with key of width/height, as index 0/1 would not tell too much info about what it indicates.